### PR TITLE
US- 06 logout branch 

### PR DESCRIPTION
**Pull request by NicosMQ**

### CODE REVIEW

**Review by Ludovicscelles — APPROVED** [review]

File: client/src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -35,6 +35,33 @@
 .searchBarDesktop {
   display: none;
 }
+
+.user-button {
```
Comment (VinchC) [inline client/src/components/NavBar/NavBar.css]: camelCase

File: client/src/components/NavBar/NavBar.css [hunk 2]
```diff
@@ -35,6 +35,33 @@
...
+
+.buttonLogout {
+  display: none;
+}
+
+.popoverContainer {
+  display: none;
+}
+
+.user-button img {
```
Comment (VinchC) [inline client/src/components/NavBar/NavBar.css]: camelCase

**Review by VinchC — CHANGES_REQUESTED** [review]

File: client/src/components/NavBar/NavBar.jsx [hunk 3]
```diff
@@ -41,12 +67,57 @@ function NavBar() {
...
+          <button
+            type="button"
+            onClick={handleUserClick}
+            className="user-button"
+          >
+            <img src={user} className="user" alt="user" />
+          </button>
+        ) : (
+          <>
+            <button type="button" className="user-button" aria-describedby={id} onClick={handleClick} >
```
Comment (VinchC) [inline client/src/components/NavBar/NavBar.jsx]: camelCase

File: client/src/components/NavBar/NavBar.jsx [hunk 4]
```diff
@@ -41,12 +67,57 @@ function NavBar() {
           <p className="presentation">Qui&nbsp;sommes&#8209;nous&nbsp;?</p>
         </div>
         {isDesktop && <SearchBar />}
-        <img src={user} className="user" alt="logo" />
+        {!auth.isLogged ? (
+          <button
+            type="button"
+            onClick={handleUserClick}
+            className="user-button"
```
Comment (VinchC) [inline client/src/components/NavBar/NavBar.jsx]: camelCase

**Review by VinchC — CHANGES_REQUESTED** [review]

**Review by VinchC — APPROVED** [review]

**Pull request merged**